### DISCUSSION TITLE
How do I group items into collapsable entries?

### DISCUSSION THREAD

**Discussion by Tander** [shedaniel/RoughlyEnoughItems#1273] [Q & A] [answered]
I'm making a modpack and saw that REI allows to group items, but I have no groups by default and can't find how create them. How do I create groups? Preferably, without using external mods.

**Comment by shedaniel** [comment, marked as answer]
It’s planned so that you can make groups yourself within REI. but currently you can only use KubeJS

**Reply by Tander** [reply]
Aww, I see, thank you for quick response! Will wait then. Don't really want to put KubeJS in the pack just for that (I do everything I need via datapacks).

**Reply by MaxNeedsSnacks** [reply]
Well, theoretically you can use anything that adds a capability to do so through an REI plugin, but KubeJS seems to be the only one exposing it rn :P
